package.json

```diff
@@ -49,22 +49,23 @@
   "devDependencies": {
     "@lavamoat/allow-scripts": "^2.0.3",
     "@metamask/auto-changelog": "^3.1.0",
-    "@metamask/eslint-config": "^9.0.0",
-    "@metamask/eslint-config-jest": "^9.0.0",
-    "@metamask/eslint-config-nodejs": "^9.0.0",
-    "@metamask/eslint-config-typescript": "^9.0.1",
+    "@metamask/eslint-config": "^11.1.0",
+    "@metamask/eslint-config-jest": "^11.1.0",
+    "@metamask/eslint-config-nodejs": "^11.1.0",
+    "@metamask/eslint-config-typescript": "^11.1.0",
```

Comment:
This config version depends upon TypeScript ~4.8, but currently 4.7.4 is used in the lockfile. Perhaps we should update the TypeScript version here too (and switch to a `~` range to prevent accidental breaking changes going forward).